packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1070,6 +1076,28 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
     markNeedsLayout();
   }
 
+  /// Space after the last character.
```

Comment:
This is a public API so you might have to give a more detailed explanation. Same as `cursorPadding`

Reply:
review the changes, please.